common-tasks/backup-restore.md

```diff
@@ -48,7 +78,32 @@ Creating a Backup
 4. When you are ready, click **Next**. Qubes will proceed to create your backup. Once the progress bar has completed, you may click **Finish**.
 
 
-Restoring from a Backup
+Restoring from a Backup (R4.0 and later)
+-----------------------
+
+1. In **Qubes VM Manager**, click **System** on the menu bar, then click **Restore Qubes from backup** in the drop-down list. This brings up the **Qubes Restore VMs** window.
+
+2. Select the source location of the backup to be restored:
+
+   - If your backup is located on a [USB mass storage device](/doc/stick-mounting/), select the device in the drop-down box next to **Device**.
```

Comment:
There is no "drop-down box next to Device", only VM selection + path. And the same file selection dialog as in backup procedure.